netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
the the lines themselves are declaration. `%bcond_without foo` defines `foo` which defaults to `on`, `%bcond_with foo` defines `foo` with default to `off`

internally `%bcond_with*` defines or undefines `with_foo` macro which are used in .spec as `%{with foo}` `%{without foo}` or `%{?with_foo}` `%{!?with_foo}`